kratos/tests/test_materials_input.py

```diff
@@ -104,6 +116,29 @@ def test_input_cpp(self):
         KratosMultiphysics.ReadMaterialsUtility(self.test_settings, self.current_model)
         self._check_results()
 
+    def test_input_with_subproperties_cpp(self):
+
+        if (missing_external_dependencies is True):
+            self.skipTest("{} is not available".format(missing_application))
+        self._prepare_test("materials_with_subproperties.json")
+        KratosMultiphysics.ReadMaterialsUtility(self.test_settings, self.current_model)
+        self._check_results_with_subproperties()
+
+    @KratosUnittest.expectedFailure
+    def test_input_with_subproperties_cpp_expected_failure(self):
+
+        if (missing_external_dependencies is True):
+            self.skipTest("{} is not available".format(missing_application))
+        self._prepare_test("materials_with_subproperties_expected_failure.json")
+        KratosMultiphysics.ReadMaterialsUtility(self.test_settings, self.current_model)
+        # An error shall be thrown while reading the input since the format is not correct
+        try:
+            with self.assertRaisesRegex(RuntimeError, "Wrong input format while reading Properties"): #ideally a more specific error message shall be devised
+                pass #the real line shall be the one below but it segfaults badly
+        except:
+            raise Exception("A segmentation fault is issued!!")

```

Comment:
hm why does this segfault?
Why no proper error is thrown?

Reply:
I am doing exactly the same that does the test_model_part_io.py for expected failure, any better idea?

Reply:
In open mp?

Reply:
I just compied the expected failure from the model_part_io test. maybe this is not necessary

Reply:
can you please check this again?

Reply:
is this still open?

Reply:
I suggest to just remove the try/except, I never had any issues with this